include/seqan3/std/iterator

```diff
@@ -217,4 +244,4 @@ concept RandomAccessIterator = BidirectionalIterator<i> &&
 
 }  // namespace std
 
-#endif
+#endif // C++17
```

Comment:
```suggestion
#endif // < C++17
```

Reply:
no, it's not "< C++17", it's "<= C++17" or "< C++20" which is "== C++17" because the C++17 is the minimum version SeqAn supports :wink